attractions/file-input/file-input.svelte

```diff
@@ -68,7 +68,13 @@
       {...$$restProps}
       on:change={e => dispatch('change', { value, nativeEvent: e })}
     />
-    <slot>{multiple ? 'select files' : 'select a file'}</slot>
+    <slot>
+      {#if multiple}
+        <slot name="select-files-message">select files</slot>
```

Comment:
question: Are you sure these slots are necessary? I'm pretty sure that the `multiple` prop is almost always decided upfront, and in that case the default slot can be used